src/tree/ContainerAppItem.ts

```diff
@@ -212,6 +212,8 @@ export async function getContainerEnvelopeWithSecrets(context: IActionContext, s
     const concreteContainerAppEnvelope = <Required<ContainerApp>>containerAppEnvelope;
     const webClient: ContainerAppsAPIClient = await createContainerAppsAPIClient([context, createSubscriptionContext(subscription)]);
 
+    // Todo: If a container app fails to provision, this list command may error with code 400
```

Comment:
Randomly ran into this issue while testing things, making note of it and will turn this into an issue for later.